.python-version

```diff
@@ -1 +1 @@
-3.7.6
+3.8.5
```

Comment:
Any reason to now use 3.8.6?

Reply:
I used 3.8.5 because I hadn't seen 3.8.6 in my available pyenv versions, updated it now though 👍